sdk/netapp/Azure.ResourceManager.NetApp/src/Generated/Models/KeyVaultProperties.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+
+namespace Azure.ResourceManager.NetApp.Models
+{
+    /// <summary> Properties of key vault. </summary>
+    public partial class KeyVaultProperties
```

Comment:
lots of service contains this name, consider to prepend the `NetApp` to `NetAppKeyVaultProperties`